scalableminds/webknossos
# Refactor Backend: Merge UserSQL and User

### URL of deployed dev instance (used for testing):
- https://refactorbackenduser.webknossos.xyz

### Steps to test:
- set up wK with multiple user accounts (try registration form also)
- check that rights/team management still works
- click around, general testing
- user viewport/dataset settings should work (be saved, loaded again after refresh)
- create some tasks, trace some, download project

### Issues:
- contributes to #2521

------
